doc/developer/03-manual-deploy.md

```diff
@@ -0,0 +1,45 @@
+# Deploying Manually to Production
+
+## Supabase Configuration
+
+To configure Supabase, please have a look at [dedicated configuration guide](./supabase-configuration.md).
```

Comment:
Following the numbered steps in the main README, this should be already done.

I you really want to keep this section I suggest a rewrite like

```suggestion
This guide assumes you have already configured your Supabase instance. Id you did not, please have a look at the [dedicated configuration guide](./01-supabase-configuration.md).
```